src/hue/rest/light_level.rs

```diff
@@ -0,0 +1,43 @@
+use color_eyre::Result;
+use serde::Deserialize;
+
+use crate::{
+    protocols::https::{mk_get_request, HyperHttpsClient},
+    settings::Settings,
+};
+
+use super::common::Owner;
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct LightLevelEventData {
+    pub light_level: i64,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct LightLevelData {
+    pub id: String,
+    pub id_v1: String,
+    pub owner: Owner,
+    pub enabled: bool,
+    pub light_level: Option<LightLevelEventData>,
```

Comment:
Possible typo here as well, should compare this to the output of the light_level API but if it's consistent with the eventsource API then this field should be `light` as well.